### ISSUE TITLE
Not clear how to install or use

### ISSUE THREAD

**Issue by JivanRoquet** [Avi-D-coder/implicit-hie#25]
Hello,

I've installed `implicit-hie` by adding the following in `stack.yaml`:

```yaml
extra-deps:
  - git: https://github.com/Avi-D-coder/implicit-hie # implicit-hie
    commit: fc1d3329148ea6616cd42d1919c5385a1a577af2
```

and in `project.cabal`:

```cabal
executable descriptions
  build-depends:       ...,
                       ...,
                       implicit-hie,
                       base >= 4.7 && < 5
```

and running `stack build`.

When I type `gen-hie` into a terminal, I get a `command not found: gen-hie` error.

Is there something specific more to do? How can we declare where the executable should go?

**Comment by jneira** [comment]
@JivanRoquet hi! you are right: basic instructions on how to install and use it would be great.

This package (implicit-hie) contains an executable (gen-hie) that can be installed and used totally independent of your package, so you dont need to add it to `stack.yaml` nor `.cabal` file.

You can install whataver excutable in the global path with stack calling `stack install whatever-package` so `stack install implicit-hie` will build and install the `gen-hie` executable. Then you can run `gen-hie` in the project root directory and it will generate a config file named `hie.yaml`. 

I would keep this open until docs are complete.

**Comment by jneira** [comment]
Closed by #35
